autogen/agentchat/contrib/gpt_assistant_agent.py

```diff
@@ -112,7 +111,13 @@ def __init__(
         # lazily create threads
         self._openai_threads = {}
         self._unread_index = defaultdict(int)
-        self.register_reply(Agent, GPTAssistantAgent._invoke_assistant)
+        self.register_reply([Agent, None], GPTAssistantAgent._invoke_assistant, position=1)
```

Comment:
```suggestion
        self.register_reply([Agent, None], GPTAssistantAgent._invoke_assistant, position=2)
```
because of a recent change #794